<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPad</class>
 <widget class="QMainWindow" name="QtPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>831</width>
      <height>571</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="separator"/>
    <addaction name="saveFile"/>
    <addaction name="saveFileAs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>Edit </string>
    </property>
    <addaction name="copy"/>
    <addaction name="paste"/>
    <addaction name="separator"/>
    <addaction name="cut"/>
    <addaction name="redo"/>
    <addaction name="undo"/>
    <addaction name="separator"/>
    <addaction name="setBackgroundColor"/>
    <addaction name="separator"/>
    <addaction name="setTextColor"/>
    <addaction name="setTextSize"/>
    <addaction name="setTextFont"/>
    <addaction name="setTextBackgroundColor"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newFile"/>
   <addaction name="openFile"/>
   <addaction name="saveFile"/>
   <addaction name="saveFileAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-copy.png</normaloff>:/Imajes/041-copy.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFile">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-export.png</normaloff>:/Imajes/041-export.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-save.png</normaloff>:/Imajes/041-save.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveFileAs">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-save-1.png</normaloff>:/Imajes/041-save-1.png</iconset>
   </property>
   <property name="text">
    <string>Save File As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="paste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="cut">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-cut.png</normaloff>:/Imajes/041-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="undo">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-undo.png</normaloff>:/Imajes/041-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="icon">
    <iconset resource="Imajes/appresources.qrc">
     <normaloff>:/Imajes/041-redo.png</normaloff>:/Imajes/041-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="setBackgroundColor">
   <property name="text">
    <string>Set Background Color</string>
   </property>
  </action>
  <action name="setTextColor">
   <property name="text">
    <string>Set Text Color</string>
   </property>
  </action>
  <action name="setTextSize">
   <property name="text">
    <string>Set Text Size</string>
   </property>
  </action>
  <action name="setTextFont">
   <property name="text">
    <string>Set Text Font</string>
   </property>
  </action>
  <action name="setTextBackgroundColor">
   <property name="text">
    <string>Set Text Background Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Imajes/appresources.qrc"/>
 </resources>
 <connections/>
</ui>
